app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="person-info">
+  <table>
+    <tr>
+      <% if @tasks[:todo].empty? && @tasks[:completed].empty? %>
+      <h3>This person has no tasks to show.</h3>
+      <% else %>
+        <th>
+          TASK
+        </th>
+        <th class="status">
+          STATUS
+        </th>
+      </tr>
+      <% @tasks[:todo].each do |task| %>
```

Comment:
Because these two `each` loops have the same HTML code within them, you might be able to condense this down to one loop over the combined sets of tasks: `(@tasks[:todo] + @tasks[:completed]).each`.